visualization/vis-master/covid-csv-tools/dist/meta.js

```diff
@@ -68,6 +68,12 @@ exports.targetFullNameCum = {
   '3-ahead': '3 wk ahead cum death',
   '4-ahead': '4 wk ahead cum death'
 };
+exports.targetFullNameIncCase = {
```

Comment:
Is this variable used anywhere?

Reply:
Yep, it's used down below to pick out the incident case data from the filtered CSV files.

Reply:
Maybe because it has `csv` in the filename?